bucket/posh-git-beta.json

```diff
@@ -0,0 +1,21 @@
+{
+    "homepage": "http://dahlbyk.github.io/posh-git/",
+    "description": "A PowerShell module which provides Git/PowerShell integration.",
+    "version": "1.0.0-beta3",
+    "license": "MIT",
+    "url": "https://github.com/dahlbyk/posh-git/archive/v1.0.0-beta3.zip",
+    "hash": "e1375fe7e66aeecc7cacc4a1c03c569839ec17312d6ebfdc39412911eb7dbb5c",
+    "extract_dir": "posh-git-1.0.0-beta3\\src",
+    "psmodule": {
+        "name": "posh-git"
+    },
+    "checkver": {
+        "url": "https://github.com/dahlbyk/posh-git/releases",
+        "regex": "\\/releases\\/tag\\/(?:v|V)?(1[\\d.]+(?:-beta\\w*)?|[\\d.]+-beta\\w*)"
```

Comment:
This would break updates for the `posh-git-beta` app (compared to `posh-git`) once version 1 releases. This catches any beta (as currently tagged) or future releases not in the current (v0.*) series, allowing for some leeway on updating the regex once v1 is stable (it'll want to be changed to v2 at that point).

Reply:
If you want to catch only `v1`'s beta, use `/releases/tag/v(1[\\d.]+-\\w+)` and rename manifest to `posh-git1-beta`. Suggested one catches all beta versions and the version may lag behind current stable one. Ref: [`python-beta`](https://github.com/ScoopInstaller/Versions/blob/master/bucket/python-beta.json)